docs/features/source-generators.cookbook.md

```diff
@@ -344,7 +344,7 @@ partial class MyRecord
 }
 ```
 
-This attribute could also be used for #participate-in-the-ide-experience,
+This attribute could also be used for [Participate in the IDE experience](#participate-in-the-ide-experience),
```

Comment:
I wasn't exactly sure if I needed to re-arrange the entire paragraph here since this makes this line much longer. There seems to be inconsistent style throughout the file where some paragraphs are trimmed at ~90 characters